kodi_addon_checker/addons/Repository.py

```diff
@@ -16,13 +16,19 @@
 
 
 class Repository():
-    def __init__(self, version, path):
+    def __init__(self, version, path, local_xml=False):
```

Comment:
can you add docstrings here, as your adding a new param (it should be documented that it's only use is testing)

Reply:
Done, I also added simple descriptive doc strings to the unit test